query/organization/org_default_repo_permission_none.sql

```diff
@@ -2,10 +2,16 @@ select
   -- Required Columns
   html_url as resource,
   case
+    when default_repo_permission is null then 'skip'
     when default_repo_permission = 'none' then 'ok'
     else 'alarm'
   end as status,
-  coalesce(name, login) || ' base permission is ' || default_repo_permission || '.' as reason,
+
+  case
+    when default_repo_permission is null then 'user is not an owner of this repo'
```

Comment:
Can you please update the reason to be similar to https://github.com/turbot/steampipe-mod-github-compliance/blob/67c01dddf393d263006a0ce4b235458be8aa45e5/query/organization/org_two_factor_required.sql#L10?